src/pages/Home/VideoCard.jsx

```diff
@@ -0,0 +1,34 @@
+import "../../styles.css";
+import "./home.css";
+
+import React from "react";
+import { Link } from "react-router-dom";
+
+import AddToPlaylist from "../../utils/AddToPlaylistBttn";
+import LikeBttn from "../../utils/LikeBttn";
+import WatchLaterButton from "../../utils/WatchLaterBttn";
+

```

Comment:
You can have a index.js in utlis and export all in that, so that all these can be imported in 1 statement and reduced path also

Reply:
this was a great suggestion, coz in index.js can only have named exports and by doing that I encountered few typos